aequilibrae/paths/traffic_assignment.py

```diff
@@ -409,7 +409,7 @@ def results(self) -> pd.DataFrame:
         res1 = assig_results[0]
 
         tot_flow = self.assignment.fw_total_flow[idx]
```

Comment:
@pedrocamargo ..... it looks like the intention was always to use the tot_flow variable in the manner that @barisdemirdelen has fixed.

I say looks like as the variable is otherwise unused - which should be a flake8 violation.... I thought you had flake8 setup as a linting stage (but i can't seem to see it in the re-shuffle to use github workflows) - I would have expected it to catch something like this.

Reply:
Sorry - that's my bad - the variable is used... my highlighting was just broken :/

Reply:
nice catch @barisdemirdelen